src/TodoList.js

```diff
@@ -1,25 +1,26 @@
 import React from 'react'
+import TodoListItem from './TodoListItem'
 
-const todoList = [
-    {
-        id: 1,
-        title: 'Study',
-    },
-    {
-        id: 2,
-        title: 'Do Homework',
-    },
-    {
-        id: 3,
-        title: 'Turn in assignment',
-    }
-]
+// const todoList = [
```

Comment:
Go ahead an removed the commented code.  It will be available in source control if you need to reference it